pulse/templates/includes/https/hsts-donut.html

```diff
@@ -0,0 +1,82 @@
+<script type="text/javascript">
+(function() {
```

Comment:
Correct me if this is just my ignorance talking, but instead of having `hsts-donut.html` and `https-donut.html`, can we have a single implementation of the donut generating function that takes a few parameters?

The code in both of these seems very similar.

Reply:
NO

yes

Maybe